### PR TITLE
User namespace support for the app

### PR DESCRIPTION
**Pull request by surajssd**
* Add support of user namespace. Support parameter to enable/disable user namespace for a container.
* In frontend add support to select user namespace using a drop down to the index page where user can enable or disable user namespace for a container.

### CODE REVIEW

File: docker.go [hunk 1]
```diff
@@ -133,79 +135,95 @@ func NewContainerHostConfig(opts ...hostOptions) *container.HostConfig {
 
 }
 
+func seccompProfile(userns bool) string {
+	if userns {
+		return seccompWeakProfile
```
Comment (alban) [inline docker.go]: The seccomp weak profile should be orthogonal to docker using a new userns for the containers. We had 3 basic profiles:

- default docker (with default seccomp, without docker creating a new userns)
- weak docker (with weak seccomp (allowing the container to create new userns), without docker creating a new userns)
- docker + userns (with default seccomp, with docker creating a new userns)

Comment (surajssd) [reply docker.go]: That means I can get rid of https://github.com/kinvolk/contained.af/pull/8/commits/58d7ddd08bb022d4f6bce5ad5681989b48b96799 ?

Comment (alban) [reply docker.go]: I think yes

**Review by pothos — APPROVED** [review]
Works as observed by checking the `/proc/1/uid_map` content.

**Pull request merged**